atm.js

```diff
@@ -1,21 +1,89 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+"use strict";
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+// var checkingBalance = document.querySelector("div#checking_balance");
+// checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "")
 
-}
+// User chooses either the checking or savings account.
 
-function checkingWithdrawal(amount){
+$(document).ready(function() {
+	console.log("Bank is ready.");
 
-}
+	// Add an event to checking's deposit button.
+	// User should be abe to deposit money to checking.
+	// User should be able to view current balance.
+	// User should be able to view updated balance.
 
-function savingsDeposit(amount){
+	$("#addToChecking").on("click", function() {
 
-}
+		var depositMoneyToChecking = parseInt($("#checkingInput").val());
+		var checkingBalance = parseInt($("#checkingBalance").text().replace("$", "")); // Had lots of help here since innerHTML wasn't working for me.
+		var newCheckingBalance = "$" + (checkingBalance + depositMoneyToChecking);
+		
+		$("#checkingBalance").text(newCheckingBalance);
+			console.log("Added $" + depositMoneyToChecking + " to the checking account.");
```

Comment:
make sure that your indentation is consistent (in general you're doing pretty well, but your console.log lines are indented too far).